### PR TITLE
Itsmekori/eng 507 build tetris integration

### PR DESCRIPTION
**Pull request by IvySwing**
migrated files from local to redwood branch and test to see if it works as a component.

### CODE REVIEW

File: web/src/components/TetrisGameApp/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import TetrisApp from './App'
+
+const rootElement = document.getElementById('root')
+ReactDOM.render(
+  <StrictMode>
+    <TetrisApp />
+  </StrictMode>,
+  rootElement
+)
```
Comment (stormcloud266) [inline web/src/components/TetrisGameApp/index.js]: We won't need this file now that it's a component and not a stand alone app

File: web/src/components/TetrisGameApp/components/GameController.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.GameController {
```
Comment (stormcloud266) [inline web/src/components/TetrisGameApp/components/GameController.css]: Just a note for later, this is why it keeps jumping to the top of the page. It's autofocusing on the input which is positioned -100em above the game.

File: web/src/components/TetrisGameApp/components/Menu.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  background: #3fccbb;
+  border-radius: 20px;
+  border: none;
+  box-shadow: rgba(0, 0, 0, 0.5) 0px 0px 60px;
+  cursor: pointer;
+}
+
+.Instructions {
+  line-height: 20px;
+  border-style: solid;
```
Comment (stormcloud266) [inline web/src/components/TetrisGameApp/components/Menu.css]: Let's do a little padding here just so the text isn't right up against the border

Screenshot 2022-12-15 at 10 23 25 AM

**Review by stormcloud266 — CHANGES_REQUESTED** [review]
This is really rad, just a couple small things

File: web/src/components/TetrisGameApp/index.js (hunk 1 above)
Comment (IvySwing) [reply web/src/components/TetrisGameApp/index.js]: @stormcloud266 file has been deleted

File: web/src/components/TetrisGameApp/components/GameController.css (hunk 2 above)
Comment (IvySwing) [reply web/src/components/TetrisGameApp/components/GameController.css]: @stormcloud266 I made a comment in the GameController.css file to reference the issue with the jump to the top of page

File: web/src/components/TetrisGameApp/components/Menu.css (hunk 3 above)
Comment (IvySwing) [reply web/src/components/TetrisGameApp/components/Menu.css]: image

@stormcloud266 let me know your thoughts

**Review by stormcloud266 — APPROVED** [review]
Awesome

File: web/src/components/TetrisGameApp/components/Menu.css (hunk 3 above)
Comment (stormcloud266) [reply web/src/components/TetrisGameApp/components/Menu.css]: looks good!

**Pull request merged**